packages/client/wallets/aa/front/components/SignMessages.tsx

```diff
@@ -54,6 +56,7 @@ const SignComponent: React.FC<SignComponentProps> = ({ aaWallet }) => {
                         { name: "enableData", type: "bytes" },
                     ],
                 },
+                account: await aaWallet.getAddress(),
```

Comment:
Added to account for update in `packages/client/wallets/aa/front/services/walletService.ts`